soapui/src/main/java/com/eviware/soapui/impl/wsdl/actions/testsuite/CloneTestSuiteAction.java

```diff
@@ -60,55 +60,52 @@ public CloneTestSuiteAction() {
     }
 
     public void perform(final WsdlTestSuite testSuite, Object param) {
-        if (dialog == null) {

```

Comment:
I think we don't need dialog to be a class field any more.

Reply:
It used in the closure so we need to keep it as a class member. Closure stores a value of local reference on closure-born moment (without global changes of code of this action it will be null).